client/src/modules/Schedule/components/TableView/TableView.tsx

```diff
@@ -136,17 +145,23 @@ export function TableView({ data, settings, statusFilter }: TableViewProps) {
   );
   const columns = filteredColumns as ColumnsType<CourseScheduleItemDto>;
 
+  const handleTagClose = (removedTag: string) => {
+    setTagFilter(tagFilter.filter(t => t !== removedTag) ?? []);
```

Comment:
According to the types annotations, right part of the expression inside `filter` callback will never be called

Reply:
Agree. Fixed